mapswipe_workers/mapswipe_workers/firebase_to_postgres/transfer_results.py

```diff
@@ -82,15 +82,30 @@ def transfer_results_for_project(project_id, results, filter_mode: bool = False)
         # The user_id is used as a key in the postgres database for the results
         # and thus users need to be inserted before results get inserted.
         results_user_id_list = get_user_ids_from_results(results)
+        results_user_group_id_list = set(
```

Comment:
Can you add a return type annotation to the functions you're modifying, so that `mypy` will check them?

Reply:
Not sure what you meant here.